dail/postponed-expedited-snap-verifications.vbs

```diff
@@ -124,7 +263,9 @@ End If
 case_note_header = "--- SNAP Closed " & closure_date & " - Expedited Autoclose ---"
 call write_variable_in_case_note(case_note_header)
 call write_bullet_and_variable_in_case_note("Reason for closure", "Delayed verifications were not submitted and Expedited SNAP Autoclosed.")
-If verifs_needed <> "" then call write_bullet_and_variable_in_case_note("Verifs needed", verifs_needed)
+If verifs_needed <> "" then call write_bullet_and_variable_in_case_note("Verifications needed", verifs_needed)
+
+'To do - check on the case noting intake notes
```

Comment:
This part can be removed. It was part of the statewide collaborative different county supports and we don't need it any more.

Reply:
Removed this line of code related to SNAP REIN date